modules/gdscript/gdscript_parser.h

```diff
@@ -209,7 +209,7 @@ class GDScriptParser {
 
 		bool _static;
 		MultiplayerAPI::RPCMode rpc_mode;
-		bool has_yield;
+		bool has_yield_or_await;
```

Comment:
```suggestion
		bool has_coroutine;
```
Just a small suggestion.

Reply:
I like that. I'll have to change that over.